<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кулинарная книга</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="placeholderText">
         <string>Поиск рецептов...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="tagFilterComboBox">
        <property name="placeholderText">
         <string>Фильтр по тегам</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="leftPanel" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="recipesListWidget">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="addButton">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editButton">
             <property name="text">
              <string>Редактировать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="recipeNameLabel">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Название рецепта</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="categoryLabel">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Категория</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>Описание рецепта</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="cookingTimeLabel">
              <property name="text">
               <string>Время приготовления</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="difficultyLabel">
              <property name="text">
               <string>Сложность</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="tagsLabel">
            <property name="text">
             <string>Теги</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Ингредиенты:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ingredientsTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="maximumHeight">
             <number>150</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Шаги приготовления:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="stepsTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="minimumHeight">
             <number>200</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionAddRecipe"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddRecipe">
   <property name="text">
    <string>Добавить рецепт</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>